submissions/yanyshpolska/dom/js/script.js

```diff
@@ -0,0 +1,72 @@
+    image: "./img/black.jpg",
+  },
+  {
+    type: "Green Tea",
+    description:
+      "Green tea production endeavors to avoid the oxidation of the tea leaves, in order to retain its natural green color and fresh flavor. In Japan, the leaves are steamed, while other countries will pan-fire or dry it through other methods. This type of tea has a more delicate flavor than black tea and often brews up pale green or golden in color.",
+    image: "./img/green.jpg",
+  },
+  {
+    type: "Mate",
+    description:
+      "Mate tea is made from the yerba mate a South American holly tree similar to the tea plant. It’s not strictly “tea” as it’s not made from the tea plant, Camelia Sinensis, but it has similar benefits to tea since it’s similarly high in antioxidants and contains caffeine. In South America it’s drunk from a hollow gourd with loose leaves, hot water and a metal strainer straw.",
+    image: "./img/mate.jpg",
+  },
+  {
+    type: "Mint Tea",
+    description:
+      "Mint tea is a herbal tea made by infusing mint leaves in hot water. Mint tea made with peppermint leaves is called peppermint tea, and mint tea made with spearmint is called spearmint tea. There also exist teas that infuse peppermint and spearmint leaves.",
+    image: "./img/mint.jpg",
+  },
+  {
+    type: "Rooibos",
+    description:
+      "Rooibos tea is also known as red tea or red bush tea. It is made using leaves from a shrub called Aspalathus linearis, usually grown on the western coast of South Africa. Rooibos is a herbal tea and is not related to green or black tea.",
+    image: "./img/rooibos.jpg",
+  },
+  {
+    type: "Pu-erh",
+    description:
+      "Pu-erh (also spelled pu'er) comes exclusively from China and is famous for its distinctively earthy flavor. Pu-erh is tea that has been fermented, often stored underground for several years. Traditionally, pu-erh is compressed into round cakes and can be very expensive.",
+    image: "./img/puer.jpg",
+  },
+];
+
+const createContent = function (objectTea) {
+  const wholeArticle = document.querySelector("article.more");
+  wholeArticle.innerHTML = "";
+
+  const picture = document.createElement("img");
+  picture.classList.add("more__picture");
+  picture.setAttribute("alt", `${objectTea.type}`);
+  picture.setAttribute("src", `${objectTea.image}`);
+
+  const title = document.createElement("h2");
+  title.classList.add("more__header");
+  title.innerHTML = `${objectTea.type}`;
+
+  const descr = document.createElement("div");
+  descr.classList.add("more__about");
+  descr.innerHTML = `${objectTea.description}`;
+
+  wholeArticle.append(picture);
+  wholeArticle.append(title);
+  wholeArticle.append(descr);
+};
+// ===== initial page
+createContent(teas[0]);
+
+const listOfTeasNames = document.querySelector("ul.menu__items");
+
+listOfTeasNames.onclick = function (event) {
+  const objectTea = teas.filter(
+    (obj) => obj.type === event.target.innerHTML
```

Comment:
If we have the same name in the result we get more than one type of tea.
Modernize your tea list, and add an `id` or `teaId` property. This field was unique to one type of tea. The id field you can write to a [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset) of the element. Then you can compare the value from the date and the selected element.